stock_picking_delivery_link/tests/test_delivery_link.py

```diff
@@ -26,6 +26,16 @@ def setUpClass(cls):
                 "product_id": test_carrier_product.id,
             }
         )
+        # We need to know if purchase module is installed
+        cls.purchase_installed = False
```

Comment:
Maybe it is cleaner to use 
```suggestion
        cls.purchase_installed = cls.env.ref("base.module_purchase").state == "installed"
```